examples/step-1/doc/intro.dox

```diff
@@ -128,6 +128,11 @@ command line, you may be interested in watching lectures 2.9 and 2.91.
 (@dealiiVideoLectureSeeAlso{2.9,2.91}) These give overviews over the command
 line and on what happens when compiling programs, respectively.
 
+Note that deal.II is actively developed, some functions were renamed
```

Comment:
can you write this sentence as "...developed, and in the course of this development we occasionally rename or deprecate functions or classes that are, however, still referenced in these video lectures."?